telethon/telegram_client.py

```diff
@@ -2440,17 +2440,17 @@ def list_update_handlers(self):
 
         # Add the mark to the peers if the user passed a Peer (not an int),
         # or said ID is negative. If it's negative it's been marked already.
-        # Look in the dialogs with the hope to find it.
```

Comment:
Not sure why you removed this, you're looking in the dialogs *with the hope* to find it. It might not be there. It might be someone from a group, looking in the dialogs is just a safe thing to do when you hardly have more information (what is the user's program doing exactly? Where could we possibly look for this peer othewise?).

Reply:
I interpreted the "hope" part as acknowledging that the dialogue may not be in the first 100